### PR TITLE
Add content + form on landing page and style form

### PR DESCRIPTION
**Pull request by Danielle-P**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,6 +9,7 @@
       href="https://fonts.googleapis.com/css?family=BenchNine:400,700|Raleway:400,600"
       rel="stylesheet"
     />
+    <link rel="shortcut icon" href="favicon.ico" type="image/x-icon" />
```
Comment (benadamstyles) [inline index.html]:
```suggestion
    <link rel="icon" href="favicon.ico" type="image/x-icon" />
```

File: index.html [hunk 2]
```diff
@@ -22,12 +23,60 @@ <h1>James Robertson</h1>
...
+          >
+
+          <div class="dietary-details">
+            <input
+              type="checkbox"
+              name="dietary-requirements"
+              value="dietary-requirements"
+              id="dietary-requirements"
+            />
+            <label for="dietary-requirements" class="checkbox-label"
```
Comment (benadamstyles) [inline index.html:58]: Can you wrap this?

File: reset.less [hunk 3]
```diff
@@ -34,40 +34,60 @@ body {
...
-ol,
-ul {
-  list-style: none;
+a {
+  color: @primary;
+
+  &:hover,
+  &:focus {
+    color: lighten(@primary, 10%);
+    cursor: pointer;
```
Comment (benadamstyles) [inline reset.less]:
```suggestion
```
Not necessary

File: reset.less [hunk 4]
```diff
@@ -34,40 +34,60 @@ body {
...
+  letter-spacing: 0.4rem;
+  font-size: 1.2rem;
+  margin: 1rem 0 0.5rem;
+
+  &:hover,
+  &:active,
+  &:focus {
+    outline: none;
+    background: lighten(@primary, 10%);
+    cursor: pointer;
```
Comment (benadamstyles) [inline reset.less]:
```suggestion
```
Buttons shouldn't have `cursor: pointer`

File: style.less [hunk 5]
```diff
@@ -18,9 +13,106 @@ header {
...
+      text-align: center;
+      background: @input;
+      color: lighten(@body, 2%);
+      font-size: 1rem;
+      font-family: @body-font;
+
+      &:focus {
+        outline: 1px solid darken(@input, 5%);
+
+        &::-webkit-input-placeholder {
```
Comment (benadamstyles) [inline style.less]: Are you sure autoprefixer doesn't do this for you?

File: style.less [hunk 6]
```diff
@@ -18,9 +13,106 @@ header {
...
+
+    .dietary-details {
+      label {
+        display: inline-block;
+      }
+      .dietary-text-field {
+        opacity: 0;
+        height: 0;
+        overflow: hidden;
+        transition: 0.5s;
```
Comment (benadamstyles) [inline style.less:100]:
```suggestion
        transition: opacity 0.5s;
```

**Review by benadamstyles — CHANGES_REQUESTED** [review]

File: index.html (hunk 2 above)
Comment (Danielle-P) [reply index.html:58]: Can't be done because the css expects the input to be a sibling

**Review by benadamstyles — APPROVED** [review]

**Pull request merged**